roles/common/tasks/backup.yaml

```
@@ -15,15 +52,28 @@
     exclude_path:
       - "{{ maas_exclude_backup_path }}"
     dest: "{{ maas_backup_dest_path }}"
+  when: archive_list is defined
+  timeout: 3600
 
-# ansible.builtin.fetch has the ability to be oom killed on large files, so we're using scp instead
 - name: Download Backup
-  ansible.builtin.command: scp {{ ansible_user }}@{{ inventory_hostname }}:{{ maas_backup_dest_path }} {{ maas_backup_download_path }}
-  delegate_to: localhost
-  become: false # don't need sudo locally
-  changed_when: false
+  ansible.builtin.fetch:
+    src: "{{ maas_backup_dest_path }}"
+    dest: "{{ maas_backup_download_path }}{{ maas_backup_file }}"
+  when: archive_list is defined
+  timeout: 3600
```

```suggestion
  ansible.builtin.fetch:
    src: "{{ maas_backup_dest_path }}"
    dest: "{{ maas_backup_download_path }}{{ maas_backup_file }}"
  when: archive_list is defined
  timeout: 3600
  become: false
```

Running this task with `become` can lead to OOM issues as explained [here.](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/fetch_module.html#notes)
Also there is no need for this task to run with `become`, but it happens anyways as the playbook includes the whole task file with `become`.